stl/inc/type_traits

```diff
@@ -1098,27 +1095,32 @@ struct _Aligned<_Len, _Align, double, false> {
 
 template <size_t _Len, size_t _Align>
 struct _Aligned<_Len, _Align, int, false> {
-    _NEXT_ALIGN(double);

```

Comment:
this looks like a perhaps unrelated change? how'd this get here?

Reply:
Please see the description - it is indeed an unrelated cleanup. Would you like me to separate it, which I can do trivially?

Reply:
nah, just something to keep in mind. Sorry, I skipped right to the code since you described it in standup :D.